backend/src/packages/chaiNNer_standard/image_utility/compositing/z_stack.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import annotations
+
+from enum import Enum
+from typing import List
+
+import numpy as np
+
+from nodes.group import group
+from nodes.properties.inputs import EnumInput, ImageInput
+from nodes.properties.outputs import ImageOutput
+from nodes.utils.utils import ALPHABET, get_h_w_c
+
+from .. import compositing_group
+
+class Expression(Enum):
+    MEDIAN = "median"
+    MEAN = "mean"
```

Comment:
Which [mean](https://en.wikipedia.org/wiki/Mean#Types_of_means) is it?

Reply:
Arithmetic. Should be the same results as if you had blended n layers with 1/n opacity each (with most transparent at the top).